packages/fetchai/skills/confirmation_aw2/README.md

```diff
@@ -10,11 +10,13 @@ This skill searches for an agent in a vicinity on the SOEF that sells the data t
 ## Behaviours
 
 * `search`: searches for data selling service on SOEF
+* `transaction`: sequentially processes transactions' settlements on a blockchain
 
 ## Handlers
 
+* `default`: handles `default` messages for registration
 * `fipa`: handles `fipa` messages for negotiation
-* `ledger_api`: handles `ledger_api` messages for payment
+* `ledger_api`: handles `ledger_api` messages for processing transaction settlements
```

Comment:
ledger_api is generally for all interactions with ledger...

Reply:
ok yes, I'll keep it generic